### PR TITLE
fix(storages.components.clipboard): add missed tooltip feature

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-9093`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-9655
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
1. display tooltip `copied` message once clipboard is copied
2. reset tooltip message once clipboard is clicked
3. add missed semicolon for some content key

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js [hunk 1]
```diff
@@ -22,4 +46,25 @@ export default class PasswordClipboardComponentController {
...
     );
   }
+
+  reset() {
+    this.model.isTextCopied = false;
+    this.model.tooltipText = this.getClipboardTooltipContent();
+  }
+
+  onClipboardClick() {
+    this.reset();
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js:56]:
```suggestion
    return this.reset();
```

File: packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js [hunk 2]
```diff
@@ -2,16 +2,40 @@ import ClipboardJS from 'clipboard';
...
+      };
+
+      this.model.clipboard
+        .on('success', (evt) => this.onClipboardCopySuccess(evt))
+        .on('error', (evt) => this.onClipboardCopyFail(evt));
+    });
+  }
+
+  $onDestroy() {
+    this.model.clipboard.destroy();
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js:31]:
```suggestion
    return this.model.clipboard.destroy();
```

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js:56]: done

File: packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js:31]: done

**Comment by sachinrameshn** [comment]
> 

Description dont have FR translation point.

File: packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js [hunk 3]
```diff
@@ -22,4 +46,25 @@ export default class PasswordClipboardComponentController {
...
     );
   }
+
+  reset() {
+    this.model.isTextCopied = false;
+    this.model.tooltipText = this.getClipboardTooltipContent();
+  }
+
+  onClipboardClick() {
+    return this.reset();
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js:56]: Can we set this.model directly here by avoiding reset() call.

**Review by sachinrameshn — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js (hunk 3 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js]: i think it's better to have a method for that.
we are more explicit and we know what the function do.

But you have right, a function with name reset means nothing.
i changed it to `resetToDefault()`.

Comment (sachinrameshn) [reply packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js]: Okay Mohammed

**Pull request merged**